### ISSUE TITLE
Strange rgb codes appear in ADE terminal when running a custom environment

### ISSUE THREAD

**Issue by bsatoriu** [MAAP-Project/ZenHub#1113]
This issue occurs when navigating back to a running workspace that is using a custom Conda environment. 

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMWdsQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--014411b1c707f0a1b0eebecb22aaa93e0d9d4d8d/Screenshot%202024-10-04%20at%201.35.22%E2%80%AFPM.png" alt="Screenshot 2024-10-04 at 1.35.22 PM.png" />

**Comment by chuckwondo** [comment]
@bsatoriu, please check your image link. Clicking the link shows "Non-Image content-type returned" in my browser. Although I cannot currently see the intended image, I suspect this is the same issue that I've encountered myself.

**Comment by bsatoriu** [comment]
That's strange @chuckwondo. I'm seeing the image from my side. Let's try this again...

<img width="1368" alt="Screenshot 2024-10-04 at 1 35 22 PM" src="https://github.com/user-attachments/assets/218853ef-265a-4888-9823-417bd9067474">

**Comment by chuckwondo** [comment]
Yep, looks good now. That is indeed what I am experiencing. Typically after switching to a different app/window/tab to do something and then returning to the ADE browser tab.
